aravis/src/manual/buffer.rs

```diff
@@ -23,8 +47,11 @@ impl Buffer {
 	/// The resulting buffer borrows the data, but it carries no lifetime.
 	/// The user has to ensure the buffer stays valid.
 	pub unsafe fn new_preallocated(data: *mut u8, len: usize) -> Self {
```

Comment:
```suggestion
	pub unsafe fn new_preallocated(data: *mut u8, len: usize) -> Self {
```

Maybe we should rename this to `new_preallocated_borrowed()`, and keep `new_preallocated()` as a deprecated alias.

Also, if we do, I think `new_preallocated_borrowed()` and `new_preallocated_owned()` sounds better than `new_borrowed_preallocated()` and `new_owned_preallocated()`. Although I don't feel very strongly on this matter ^^